src/config.yaml

```diff
@@ -42,6 +46,12 @@ options:
       Secret access key for the AWS IMA user. The user must have a policy that
       grants it privileges to upload to the S3 bucket. This value is required
       when backend='s3'.
+  azure_connection_string:
+    type: string
+    default: ""
+    description: |
```

Comment:
**suggestion (non-blocking):** 

Can you provide some example string here or a link to document where people can find the example.

Reply:
added it.